src/tokens/transformation/android/formats/fontStyle.js

```diff
@@ -11,20 +11,14 @@ const printDescription = (description) =>
 
 module.exports = ({ dictionary, platform, options = {}, file }) => {
   const fontStyles = dictionary.allTokens
-    // remove underlined
-    .filter(
-      (compositeToken) =>
-        compositeToken.original.value.textDecoration !== 'underline'
-    )
+    .filter((compositeToken) => compositeToken.type === 'custom-fontStyle')
     // create style
     .map((compositeToken) => {
       return (
-        `  <style name="${camelCase(compositeToken.name)}">\n` +
+        `  <style name="${changeCase.pascalCase(compositeToken.name.replace("font_android_", ""))}">\n` +
```

Comment:
Optional: use multiline string interpolation here so we can avoid all the concats/`\n`s